specification/gedcom-02-datatypes.md

```diff
@@ -317,6 +317,27 @@ where:
 
 The URI for the `MediaType` data type is `dcat:mediaType`.
 
+## Shortened URI
+
+A URI, possibly shorted and documented using a [documented extension tag](#extension-tags).
```

Comment:
```suggestion
A URI, possibly shortened and documented using a [documented extension tag](#extension-tags).
```

Reply:
I am not shure whether this solution leads to a way without extension tags in next minor/major. As the implementation in applications is not easy (so far they can build the complete link to the EXID directly), I would prefer a solution which remains very similar in following versions of the standard. Is there any idea how a solution without extension tags could work?
The proposed solution would work, but as the size of files is not very limiting, I see too much work for implementation compared to the temporary result.